lib/shared/navigation_helper.dart

```diff
@@ -0,0 +1,7 @@
+extension LinkWithTabInfo on String {
+  String get withTabArg {
+    final uri = Uri.parse(this);
+
+    return '$this${uri.hasQuery ? '&' : '?'}fromTab=0';
```

Comment:
This 0 should be dynamic (not hardcoded), no?

Reply:
Correct, it is easy to get the current tab index from AutoRoute 👍